docs/adminguide/hosting/index.rst

```diff
@@ -0,0 +1,265 @@
+
+  The step :bash:`TOKEN=$(openssl rand 63 | base64 --wrap=86)` could easily be replaced by a value generate by a password-generator for example.
+
+  It don't need to be encoded in ``base64`` (we encode it in the script just to have printable characters).
+
+Database configuration
+----------------------
+
+Create the file ``parsec-db.env`` with the following content:
+
+.. include:: parsec-db.env
+   :code: ini
+
+Here, we set the database url and the min & max number of connections to it.
+
+SMTP configuration
+------------------
+
+We will create the file ``parsec-email.env`` to configure the access to an SMTP server (``mailhog`` in this case).
+
+We need to set the connection informations, the sender information, in which the default language the emails are sent:
+
+.. include:: parsec-email.env
+  :code: ini
+
+S3 service configuration
+------------------------
+
+Here we set the url for the parsec backend to access the S3 like service, we create the file ``parsec-blockstore.env`` with the following content:
+
+.. include:: parsec-blockstore.env
+  :code: ini
+
+.. note::
+
+   We need to escape the ``:`` with a ``\`` when specifying the port of the service.
+
+Parsec server configuration
+---------------------------
+
+We also use the following env file (``parsec.env``) to configure the ``parsec-backend``:
+
+.. include:: parsec.env
+  :code: ini
+
+The docker-compose file
+=======================
+
+We have a `docker compose`_ file (``parsec-server.docker.yaml``) that can be used to deploy a parsec backend for testing:
+
+.. _docker compose: https://docs.docker.com/compose/
+
+.. include:: parsec-server.docker.yaml
+  :code: yaml
+
+It will setup 4 services:
+
++---------------------+----------------------------+
+| Service name        | Description                |
++=====================+============================+
+| ``parsec-postgres`` | The database PostgreSQL    |
++---------------------+----------------------------+
+| ``parsec-s3``       | The Object Storage service |
++---------------------+----------------------------+
+| ``parsec-smtp``     | Provide a mock SMTP server |
++---------------------+----------------------------+
+| ``parsec-backend``  | The parsec backend server  |
++---------------------+----------------------------+
+
+Starting the infra
+==================
+
+The docker containers can be started as follow:
+
+.. code-block:: bash
+
+  docker compose -f parsec-server.docker.yaml up
+
+Initial infra setup
+===================
+
+On the first start of the infra, we need to do a one-time configuration on the database and s3.
+
+Applying the database migration
+-------------------------------
+
+(optional) You can verify that the database is accessible with:
+
+.. code-block:: bash
+
+  set -a
+  source parsec-db.env
+  docker exec -t parsec-postgres psql 'postgresql://DB_USER:DB_PASS@0.0.0.0:5432/parsec' -c "\conninfo"
+  You are connected to database "parsec" as user "parsec" on host "0.0.0.0" at port "5432".
```

Comment:
This is the expected output, isn't it? Maybe we can prefix it with "#" to show that it must not be executed as the previous lines (it seems obvious but who knows!)  
```suggestion
  # You are connected to database "parsec" as user "parsec" on host "0.0.0.0" at port "5432".
```
Or, we can prefix the first 3 lines with the "$"

Reply:
BTW we should consider using https://sphinx-copybutton.readthedocs.io/en/latest/ 
what do you think? maybe create an issue for later?

Reply:
> BTW we should consider using https://sphinx-copybutton.readthedocs.io/en/latest/
what do you think? maybe create an issue for later?

https://github.com/Scille/parsec-cloud/pull/5550

Reply:
I've done something different to better integrate with #5550